app/[organization]/[event]/stage/[stage]/components/StageLayout.tsx

```diff
@@ -55,11 +55,11 @@ export default function StageLayout() {
     return tabs
   }
   return (
-    <div className="h-full flex flex-col w-full lg:flex-row relative lg:p-4 lg:max-h-screen">
-      <div className="h-full flex flex-col w-full lg:flex-row relative p-2 lg:gap-4">
+    <div className="h-full flex flex-col w-full lg:flex-row relative p-4 lg:max-h-screen">
+      <div className="h-full flex flex-col w-full lg:flex-row relative lg:gap-4">
         <div
           ref={stickyRef}
-          className="bg-black mb-2 lg:mb-0  p-2 md:p-4 rounded-xl sticky top-[65px] z-40 flex flex-col lg:h-full w-full box-border lg:overflow-scroll lg:w-[75%]">
```

Comment:
not sure about this, the plugin bar looks to big

Reply:
On testing the dm3 chat widget, I noticed it needs more spacing, I will adjust the spacing after we finish integrating dm3 chat